js/src/cli/actions.ts

```diff
@@ -0,0 +1,80 @@
+import chalk from "chalk";
+import { Command } from "commander";
+import client from "../sdk/client/client";
+import { getOpenAPIClient } from "../sdk/utils/config";
+import { ListActionsV2Data } from "../sdk/client";
+
+export default class ActionCommand {
+  private program: Command;
+
+  constructor(program: Command) {
+    this.program = program;
+    this.program
+      .command("actions")
+      .description("Composio Actions")
+      .option(
+        "-a, --apps <appName>",
+        "List all actions for the given apps",
+        (value, previous: string[]) => previous.concat([value]),
+        []
+      )
+      .option(
+        "--tags <tagName>",
+        "List all actions for the given tags",
+        (value, previous: string[]) => previous.concat([value]),
+        []
+      )
+      .option("--use-case", "Search for actions based on the given use case")
+      .option("--limit <limit>", "Limit the number of actions to display")
+      .option("--enabled", "Only show enabled actions")
+      .option("--copy", "Copy the action to the clipboard")
+      .action(this.handleAction.bind(this));
+  }
+
+  private async handleAction(options: {
+    apps?: string[];
+    tags?: string[];
+    useCase?: string;
+    limit?: number;
+    enabled?: boolean;
+  }): Promise<void> {
+    getOpenAPIClient();
+    const { apps = [], tags = [], useCase, limit, enabled } = options;
+    if (apps) {
+      const data: ListActionsV2Data = {
+        query: {},
+      };
+      if (data?.query) {
```

Comment:
The `if (data?.query)` check is redundant since we just created the data object with a query property. Consider simplifying this by directly setting the query parameters.